src/components/utils/MapStateContext.tsx

```diff
@@ -4,16 +4,17 @@ import { Alert } from '@material-ui/lab';
 import { usePersistedState } from './usePersistedState';
 import { DEFAULT_MAP } from '../../config';
 import { PROJECT_ID } from '../../services/project';
+import { LayerIndexAttribution } from '../LayerSwitcher/helpers/loadLayers';
 
 export interface Layer {
   type: 'basemap' | 'overlay' | 'user' | 'spacer' | 'overlayClimbing';
   name?: string;
   url?: string;
   key?: string;
   Icon?: React.FC<any>;
-  attribution?: string[]; // missing in spacer TODO refactor ugly
+  attribution?: string[] | LayerIndexAttribution; // missing in spacer TODO refactor ugly
   maxzoom?: number;
-  bbox?: number[];
+  bbox?: number[] | number[][];
```

Comment:
Why do you need this? Do you have more bboxes for one layer? If yes, perhaps create a new field called `bboxes`, as this is confusing.